pkg/acsengine/const.go

```diff
@@ -90,6 +102,12 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"dnsmasq":      "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":        "pause-amd64:3.0",
 		"windowszip":   "v1.6.0intwinnat.zip",
+		"nodestatusfreq":   "10s",
+		"nodegraceperiod":   "40s",
+		"podeviction":   "5m0s",
+		"routeperiod":   "10s",
```

Comment:
The features you're identifying above are the default values of existent features. In other words, those values will change *after* we enable backoff/rate limiting on large clusters. Because we're passing in this config as a command line option now, we have to provide values. (I'm pretty sure passing these CLI options in is not toggleable per-version.)